src/main/java/Handsome.java

```diff
@@ -0,0 +1,153 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import task.Task;
+import task.ToDo;
+/**
+ * Chatbot that is able to differentiate tasks
+ */
+public class Handsome {
+    private static final ArrayList<Task> data = new ArrayList<>();
+    public static class Deadline extends Task {
+        protected String by;
+
+        public Deadline(String des, String by) {
+            super(des.split(" ", 2)[1]);
+            this.by = by;
+        }
+
+        @Override
+        public String toString() {
+            return "[D]" + super.toString() + " (by: " + by + ")";
+        }
+    }
+
+    public static class Event extends Task {
+        protected String from;
+        protected String to;
+
+        public Event(String des, String from, String to) {
+            super(des);
+            this.from = from;
+            this.to = to;
+        }
+
+        @Override
+        public String toString() {
+            return "[E]" + super.toString() + " (from: " + from + " to: " + to + ")";
+        }
+    }
+
+    /* route the task adding through this error checker first
+    private static void toDoCheck(String des) throws HandsomeException {
+        if (des.isEmpty()) {
+            throw new HandsomeException("Hey man, give me a description to work with?");
+        }
+    } */
+
+    private static void addTask(Task task) {
+        data.add(task);
+        System.out.println("Looking productive! I have added: \n" + task.toString());
+        System.out.println("Seems like the task count is " + data.size() + ", don't overwork yourself okie?");
+    }
+    private static void printList() {
+        System.out.println("Hey, here are the tasks you have!");
+        for (int i = 0; i < data.size(); i++) {
+            System.out.println((i + 1) + "." + data.get(i).toString());
+        }
+    }
+    private static void markDone(int taskNum) {
+        data.get(taskNum).markAsDone();
+        System.out.println("Congrats, you finally finished this task!");
+        System.out.println(data.get(taskNum).toString());
+    }
+
+    private static void markUndone(int taskNum) {
+        data.get(taskNum).markAsUndone();
+        System.out.println("Hey, the task is not done but time to stop procrastinating?");
+        System.out.println(data.get(taskNum).toString());
+    }
+
```

Comment:
There is a line break here, which is inconsistent with how you deal with line breaks above. Any reason why you did it this way?